.github/workflows/create_forks.yml

```diff
@@ -0,0 +1,41 @@
+# A workflow that looks at all repos in the OpenNeuroDatasets organization
+# and creates a fork for each of them in the OpenNeuroDatasets-JSONLD organization
+# if a fork does not yet exist.
+
+name: Fork Repositories
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Run every day at midnight
+
+jobs:
+  joblist:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Make forks
+        run: |
+          get_repos() {
+            local organization=$1
+            query="{
+              organization(login: \"${organization}\") {
+                repositories {
+                  totalCount
+                }
+              }
+            }"
+            nRepos=$(gh api graphql -f query="$query" | jq -r '.data.organization.repositories.totalCount')
+            gh repo list "${organization}" --limit "$nRepos" --json name --jq '.[].name'
```

Comment:
Question: do we want to explicitly exclude the `.github` repo here? I know that both orgs currently have it so it shouldn't be a problem - but in case we need to recreate the org, it would make sense for .github not to be forked.

Reply:
ah. Yeah, good idea.